client/src/components/Edit.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { useEffect, useState } from 'react'
```

Comment:
I noticed `useEffect` was brought in here but never used through various alerts in the browser and terminal consoles.